pegasus/router.rb

```diff
@@ -284,6 +284,11 @@ def update_actionview_assigns
       response.headers['X-Frame-Options'] = 'ALLOWALL'
     end
 
+    if @header['embeddable']
+      response.headers['X-Frame-Options'] = 'ALLOWALL'
+      response.headers['Content-Security-Policy'] = 'frame-ancestors *'
```

Comment:
@unlox775 I realize this is super permissive, but this particular page has no user info (it's a JS widget). I also don't actually know where this will be embedded at this point so an allow-list would be annoying to maintain. Thoughts? I feel more okay with this in Pegasus in general as the vast majority of Pegasus is static content and user agnostic

Reply:
I agree on both counts, Pegasus is a lower risk in general, and the mechanism you have here does allow for a simple allow list pattern.  Looks great!